src/server/config.rs

```diff
@@ -11,8 +7,6 @@ pub struct Config {
     /// The number of threads to spawn for handling connections. Each thread is used for one
     /// connection at a time.
     pub connection_handler_threads: usize,
-    /// Config for TLS encryption to enable HTTPS. If this is not set then normal HTTP will be used.
-    pub tls_config: Option<Arc<ServerConfig>>,
     /// The router used for handling requests.
     pub router: Router,
 }
```

Comment:
I probably wouldn't make this it's own file, but idk if you're planning on adding anything else. Either way it doesn't really matter

Reply:
Yeah I thought the config would be bigger so that's why I put it in a separate file